packages/dev/core/src/Animations/animationGroup.ts

```diff
@@ -920,6 +920,10 @@ export class AnimationGroup implements IDisposable {
             if (!skipOnAnimationEnd) {
                 this.onAnimationGroupEndObservable.notifyObservers(this);
             }
+            if (this._animatables.length > 0) {
+                // explicitly clear pending animatables when mask is applied
+                this._animatables = [];

```

Comment:
irrelevant to whether or not this is correct here, please clear using `this._animatables.length = 0` to avoid creating a new object

Reply:
```suggestion
                this._animatables.length = 0;
```

Reply:
Given the comment from @RaananW, you can simply replace the 4 lines with `this._animatables.length = 0;`